### DISCUSSION TITLE
Python Not Detected during install. Python is installed.

### DISCUSSION THREAD

**Discussion by DazzaJay** [Taxel/PlexTraktSync#1829] [Support]
Trying to install PlexTraktSync with the included setup.bat and am running into the issue "Download and install Python, then restart this project."

The only issue is, Python 3.12.2 is installed. (See near bottom of screenshot)

How do i get past this roadblock?

![Screenshot 2024-02-24 193906](https://github.com/Taxel/PlexTraktSync/assets/3713158/312a0b8d-e00d-45b3-9d6d-9dd008dc40fd)

**Comment by glensc** [comment]
you should just install PlexTraktSync with pipx

**Reply by DazzaJay** [reply]
what even is pipx, and why do i need something else to use something that should just work on its own?

**Reply by glensc** [reply]
please see project readme. it has all needed links

why pipx you can also find out from your favourite search engine.

**Reply by glensc** [reply]
also, the windows install you tried, is described in the project readme as "not supported". it was contributed by one user. we don't support that (I don't even have windows)

**Reply by DazzaJay** [reply]
So, if it doesn't work, and it's not supported, why include it in both the download package, and the install instructions?

Surley if it wasn't supported, you'd remove it from the install instructions, and the download package?

Either way, I can't get it to work in pip either. I just get "unsupported command"

**Reply by glensc** [reply]
It's community contribution, it might work and it might not work.

Can't support barely on your two words `"unsupported command"`, you need to provide more details.
